package.json

```diff
@@ -75,7 +75,7 @@
       "@examples/next-websockets-starter>@prisma/client": "https://registry.npmjs.com/@prisma/client/-/client-4.9.0.tgz?id=%40examples%2Fnext-websockets-starter",
       "@examples/trpc-next-prisma-todomvc>@prisma/client": "https://registry.npmjs.com/@prisma/client/-/client-4.9.0.tgz?id=%40examples%2Ftrpc-next-prisma-todomvc",
       "@examples/legacy-next-starter>@prisma/client": "https://registry.npmjs.com/@prisma/client/-/client-4.9.0.tgz?id=%40examples%2Flegacy-next-starter",
-      "esbuild": "0.16.3"
+      "esbuild": "0.16.17"
```

Comment:
I think we should be able to remove this override, I've added it to fix a parsing error in the types (https://github.com/evanw/esbuild/commit/9ce5f94a49ff37635dfc221477640dfee11e036b)